### PR TITLE
first test commit

### PR DESCRIPTION
**Pull request by MeghaFulaware**
practice test

### CODE REVIEW

File: Java_learning/src/gitTest/Basicgit.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package gitTest;
+
+public class Basicgit {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```
Comment (surajitsarkarofficial) [inline Java_learning/src/gitTest/Basicgit.java:6]: remove commented line

**Review by surajitsarkarofficial — CHANGES_REQUESTED** [review]
Please implement changes suggested

File: Java_learning/src/gitTest/Basicgit.java (hunk 1 above)
Comment (MeghaFulaware) [reply Java_learning/src/gitTest/Basicgit.java]: changes implemented.

**Review by surajitsarkarofficial — APPROVED** [review]
Looks good

**Pull request merged**